src/it/scala/ie/zalando/fabric/gateway/TestUtils.scala

```diff
@@ -43,9 +43,6 @@ object TestUtils {
     case object InvalidRequest extends ResourcePayload {
       def payload: String = Source.fromResource("synch/invalid.json").mkString
     }
-    case object Spp3240Payload extends ResourcePayload {
-      def payload: String = Source.fromResource("synch/spp3240OwnershipConfirmationDeserializationBug.json").mkString
-    }
```

Comment:
unrelated change. This wasn't used anywhere

Reply:
Maybe it was meant to be used in the  `SPP3240BugVerificationSpec.scala` test that was deleted?

Reply:
It wasn't used in in there, that test was using the more common Gateway Spec which is tested repeatedly elsewhere

Reply:
I meant, the names are the same, possibly it was a typo that it was using the happy path input